### DISCUSSION TITLE
Checking status of upload

### DISCUSSION THREAD

**Discussion by tomasgiden** [Azure/azure-storage-fuse#1186] [Q&A]
I am planning to use Blobfuse2 in AKS in a data processing pipeline. The job will write 50-1000 GB of data to a mount point and then exit. Then the node will scale down immediately to save money (expensive GPU). I am afraid that the node will stop before all data has been uploaded to blob storage.

Can I either:
1. Configure it so that the node does not stop before everything is in sync?
2. Check sync status inside the pod to stop the job from finishing before everything is in sync?

/Tomas

**Comment by bhansalivikas** [comment]
Once data transfer is done from your application side, you can give it a breather and try to unmount. If unmount fails then you need to wait. On successful unmount you can bring down the node.
Another way to check for transfer complete is to set file-cache-timeout=0 and then keep monitoring your configured disk-cache directory. After the transfer when the disk-cache is empty you can confirm that transfers are complete from blobfuse end as well and then unmount and bring down the node.

**Reply by tomasgiden** [reply]
If I follow any of the suggestions in here https://github.com/kubernetes-sigs/blob-csi-driver/blob/master/deploy/example/e2e_usage.md, to statically provision an existing storage account into a pvc mount in a pod, will it automatically unmount after the container stops so that the node does not spin down before the data is synced?

**Reply by bhansalivikas** [reply]
@andyzhangx  : can you answer the last part here.

**Reply by tomasgiden** [reply]
Thank you for sending the question to the right person.
